crates/sui-protocol-config/src/lib.rs

```diff
@@ -2202,6 +2203,9 @@ impl ProtocolConfig {
 
                     // Enable resharing at same initial version
                     cfg.feature_flags.reshare_at_same_initial_version = true;
+
+                    // Use tonic networking for Mysticeti.
+                    cfg.feature_flags.consensus_network = ConsensusNetwork::Tonic;
```

Comment:
Protocol version 46 has been already deployed in testnet - is it ok to go and change now the flag values?

Reply:
Good catch! I didn't realize version 46 got picked into the 1.25 release.